src/main/scala/it/pagopa/interop/backendforfrontend/service/impl/CatalogProcessServiceImpl.scala

```diff
@@ -64,4 +70,16 @@ class CatalogProcessServiceImpl(catalogProcessUrl: String, blockingEc: Execution
       invoker.invoke(request, s"Retrieving EService for $eServiceId from Catalog Process")
     }
 
+  override def updateEServiceById(eServiceId: String, updateEServiceSeed: UpdateEServiceSeed)(implicit
+    contexts: Seq[(String, String)]
+  ): Future[EService] = withHeaders { (bearerToken, correlationId, ip) =>
+    val request: ApiRequest[EService] =
+      api.updateEServiceById(
+        xCorrelationId = correlationId,
+        eServiceId = eServiceId.toString,
+        updateEServiceSeed = updateEServiceSeed,
+        xForwardedFor = ip
+      )(BearerToken(bearerToken))
+    invoker.invoke(request, s"Updating EService with $eServiceId")
```

Comment:
@galales won't this require a bit of error handling?

Reply:
the BFF is a bit different about the error handling, it forwards everything it receives, http status included.
In this service it is safe to not handle each error response 👍